rule_schema_v2.atd

```diff
@@ -430,11 +430,19 @@ type metavariable_cond = {
 
 type mvar = string
 
```

Comment:
this TODO comment is still valid? I can see an adapter below.

Reply:
good catch, thx